### DISCUSSION TITLE
Discord channel

### DISCUSSION THREAD

**Discussion by wmxsteel** [StateSmith/StateSmith#71] [General]
Was wondering if you had a discord?

There are some ideas I want to run by you.

**Comment by adamfk** [comment]
I'll set one up in a couple days and report back here. Happy to discuss!

**Comment by adamfk** [comment]
@wmxsteel here's the discord server link: https://discord.gg/rNxNGQXWsU

I probably won't be able to do a voice call until Jan 6th though.

I'm pretty new to discord. Suggestions welcome.
